.gitignore

```diff
@@ -5,5 +5,8 @@ node_modules
 .DS_Store
 
 # These files are generated from templating and should not be committed to this repo
-sample-apps/*/extensions/*/input.graphql
-sample-apps/*/web/frontend/metafields.js
\ No newline at end of file
+**/input.graphql
+**/Cargo.lock
+**/Cargo.toml
+**/shopify.function.extension.toml
```

Comment:
I think we still want `input.graphql`, `Cargo.toml`, and `shopify.function.extension.toml` in the `sample-apps/` directory right?